Dockerfile

```diff
@@ -60,6 +61,53 @@ ENV METASPLOIT_GROUP=metasploit
 # used for the copy command
 RUN addgroup -S $METASPLOIT_GROUP
 
+# -removed mingw-w64-gcc , added gcc musl-dev python3-dev libffi-dev gcompat#
+RUN apk add --no-cache bash sqlite-libs nmap nmap-scripts nmap-nselibs \
+    postgresql-libs python3 py3-pip ncurses libcap su-exec alpine-sdk \
+    openssl-dev nasm gcc musl-dev python3-dev libffi-dev gcompat
+# -- #
+RUN /usr/sbin/setcap cap_net_raw,cap_net_bind_service=+eip $(which ruby)
+RUN /usr/sbin/setcap cap_net_raw,cap_net_bind_service=+eip $(which nmap)
+
+COPY --from=builder /usr/local/bundle /usr/local/bundle
+RUN chown -R root:metasploit /usr/local/bundle
+COPY . $APP_HOME/
+COPY --from=builder $TOOLS_HOME $TOOLS_HOME
+RUN chown -R root:metasploit $APP_HOME/
+RUN chmod 664 $APP_HOME/Gemfile.lock
+RUN gem update --system
+# -- #
+RUN gem install nokogiri 
+# -- #
+RUN cp -f $APP_HOME/docker/database.yml $APP_HOME/config/database.yml
+RUN curl -L -O https://raw.githubusercontent.com/pypa/get-pip/f84b65709d4b20221b7dbee900dbf9985a81b5d4/public/get-pip.py && python3 get-pip.py && rm get-pip.py
```

Comment:
Is this necessary? It appears you're installing `py3-pip` above on line 66.

Reply:
I am not very sure what you are referring to, because the package **py3-pip** already exists on the official Dockerfile. But i added the package the package **python3-dev** because there is an issue with the impacket python library. 
![image](https://github.com/rapid7/metasploit-framework/assets/24854891/3cf8e498-64c5-480f-b13b-04f798233e34)